tests/sentry/integrations/test_metric_alerts.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+import datetime
+from django.utils import timezone
+
+from sentry.integrations.metric_alerts import incident_attachment_info
+from sentry.testutils import TestCase
+
+
+class IncidentAttachmentInfoTest(TestCase):
+    def test_correct_info_returned(self):
+        alert_rule = self.create_alert_rule()
+        date_started = datetime.datetime(2020, 8, 17, 18, 13, 55, 133221, tzinfo=timezone.utc)
+        incident = self.create_incident(
```

Comment:
@NisanthanNanthakumar I think we should have a test case if the incident does not exist since we handle that differently

Reply:
@scefali if there is no incident, I don't think the Alert Rule gets triggered?

Reply:
@NisanthanNanthakumar Sorry, i meant the `IncidentTrigger` which we query in the function

Reply:
@scefali ahh I see!! cool will make a new test for that case.